app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -89,20 +90,28 @@ def continuous_output_providers
         super << self
       end
 
+      # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
       def fill_continuous_output(continuous_output)
-        fill_planning_errors_to_continuous_output(continuous_output) unless exit_status
+        final_timestamp = (continuous_output.last_timestamp || task.ended_at).to_f + 1
+
+        if task.state == 'stopped' && task.result == 'cancelled'
+          continuous_output.add_output(_('Job cancelled by user'), 'debug', final_timestamp)
+        else
+          fill_planning_errors_to_continuous_output(continuous_output) unless exit_status
+        end
+
```

Comment:
Please add this code after the line 103-105, to get the error messages about task cancelling to the end of the output:

```ruby
        delegated_output.fetch('result', []).each do |raw_output|
          continuous_output.add_raw_output(raw_output)
        end

        final_timestamp = (continuous_output.last_timestamp || task.ended_at).to_f + 1

        if task.state == 'stopped' && task.result == 'cancelled'
          continuous_output.add_output(_('Job cancelled by user'), 'debug', final_timestamp)
        else
          fill_planning_errors_to_continuous_output(continuous_output) unless exit_status
        end
```